tests/yast2_gui/yast2_firewall.pm

```diff
@@ -18,6 +18,7 @@ use version_utils qw(is_sle is_leap is_tumbleweed);
 use yast2_shortcuts '%fw';
 use network_utils 'iface';
 use yast2_widget_utils 'change_service_configuration';
+use YaST::Module;
```

Comment:
do we need a new test? are we sure this yast2_firewall is not used where we don't have libyui-rest-api? We don't need logic for old firewall technology when using libyui.

Reply:
new test case created